### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by DJRHails**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (matt-malarkey) [inline LICENSE:1]: Any thoughts on MIT vs. GNU to consider?

**Comment by matt-malarkey** [comment]
I think if you make it .txt GitHub will detect it and make it look nicer as well

**Review by matt-malarkey — CHANGES_REQUESTED** [review]
I think if you make it .txt GitHub will detect it and make it look nicer as well

File: LICENSE (hunk 1 above)
Comment (DJRHails) [reply LICENSE:1]: Pros and Cons? MIT is the one I'm most familiar with

**Comment by DJRHails** [comment]
> I think if you make it .txt GitHub will detect it and make it look nicer as well

This actually is the automatic GitHub generated one - it says it will only recognise `README.md` and `README`

File: LICENSE (hunk 1 above)
Comment (matt-malarkey) [reply LICENSE:1]: GNU makes sure if other people distribute it it's still open source

Comment (matt-malarkey) [reply LICENSE:1]: But also should it be open source???

Comment (DJRHails) [reply LICENSE:1]: Has to be if we want all the freebies for github (e.g. organisations). DApps in general are opensource and its security by obscurity if we tried to hide the code.

**Review by matt-malarkey — APPROVED** [review]

**Pull request merged**